src/D2L.CodeStyle.Analyzers/Immutability/ImmutabilityContext.cs

```diff
@@ -214,6 +214,10 @@ ITypeSymbol type
 				return ImmutableTypeKind.Total;
 			}
 
+			if( Attributes.Objects.ConditionallyImmutable.IsDefined( type ) ) {

```

Comment:
ConditionallyImmutable really isn't any different than Immutable, other than the name and some restrictions we'll enforce down the road